string cropped_image_path
---
sensor_msgs/Image image